index.js

```diff
@@ -2312,11 +2312,41 @@ function measureText(font, text) {
 
 /**
  * Writes the image to a file
- * @param path a path to the destination file (either PNG or JPEG)
- * @param (optional) cb a function to call when the image is saved to disk
- * @returns this for chaining of methods
+ *
+ * @param {string} path a path to the destination file (either PNG or JPEG)
+ * @param {null|function} cb a function to call when the image is saved to disk
+ * @param {boolean} returnPromise
+ * @returns {Jimp|Promise} if returnPromise is true then returns a Promise otherwise `this` for chaining methods
+ */
+Jimp.prototype.write = function (path, cb, returnPromise) {
```

Comment:
@oliver-moran  asked me to help out with the PR backlog.  I'd say that since converting Node-style callback APIs into Promises is fairly straightforward for those who want it

http://exploringjs.com/es6/ch_promises.html#sec_nodejs-callbacks-promises

there's no real reason to break backward compat for it.  I'll close this out but of course if Oliver feels differently he can revive it :)